### PR TITLE
Adding greeting in kiswahili

### PR DESCRIPTION
**Pull request by peterochodo**
Hey @inca-buildmaster what do you think?

### CODE REVIEW

File: hello.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package main
+import "fmt"
+func main() {
+
+fmt.Println("Habari gani?")
```
Comment (inca-buildmaster) [inline hello.go]: also add a greeting in French

**Review by inca-buildmaster — CHANGES_REQUESTED** [review]
make sure you run 'go fmt' on the source code

File: hello.go [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package main
+
 import "fmt"
+
 func main() {
 
-fmt.Println("Habari gani?")
+	fmt.Println("Habari gani?")
+	fmt.Println("Bonne chance!")
```
Comment (inca-buildmaster) [inline hello.go]: This is not a french greeting.

**Review by inca-buildmaster — CHANGES_REQUESTED** [review]

File: hello.go (hunk 2 above)
Comment (peterochodo) [reply hello.go]: fixed

**Review by inca-buildmaster — APPROVED** [review]

**Pull request merged**